pipelinewise/cli/schemas/tap.json

```diff
@@ -22,6 +22,15 @@
         "target_schema"
       ]
     },
+    "db_conn": {
+      "type": "object",
+      "properties": {
+        "fastsync_parallelism": {
+          "type": "integer",
+          "minimum": 1
```

Comment:
can we have a realistic max value here?

Reply:
Like what?
`multiprocessing.Pool` does accept 1 as the min size of the pool.

Reply:
Setting `fastsync_parallelism` to 1000000 likely will cause a death on the host. Do we want to control it by setting a realistic max allowed value in the config, like 100 or we don't want to care and we trust the user and the OS?

Reply:
Wdyt about programmatically falling back to the number of available CPU cores if the given `fastsync_parallelism` is greater than the # cup cores?

Reply:
sounds good to me

Reply:
Made few updates:
* set maximum to 1000 ..we could re-consider it if anyone every uses a super powerful node and want to use 1000 cores. :P
* choose min between cpu cores and fastsync_parallelism

Reply:
🙇